switches/http2/http/HttpDispatcher.h

```diff
@@ -98,6 +99,9 @@ namespace Aseba
 		typedef std::map<HttpMethod, URIHandlerMap> HandlersMap;
 		//! Handlers for known method + URI couples
 		HandlersMap handlers;
+		//! Map of method, split URI to JSON-formatted OAS documentation
+		typedef std::map<HttpMethod, std::map<strings, json>> ApidocsMap;
```

Comment:
Assuming that nobody every uses the same handler with more than one method or wants to register uriPath aliases (registerHandler allows this)...